pkg/drivers/kic/kic.go

```diff
@@ -112,6 +113,28 @@ func (d *Driver) Create() error {
 		}
 	}
 
+	if err := oci.SetupContainerNode(params); err != nil {
+		return errors.Wrap(err, "setting up container node")
+	}
+
+	var waitForPreload sync.WaitGroup
+	waitForPreload.Add(1)
+	go func() {
+		defer waitForPreload.Done()
+		// If preload doesn't exist, don't bother extracting tarball to volume
+		if !download.PreloadExists(d.NodeConfig.KubernetesVersion, d.NodeConfig.ContainerRuntime) {
+			return
+		}
+		t := time.Now()
+		glog.Infof("Starting extracting preloaded images to volume")
+		// Extract preloaded images to container
+		if err := oci.ExtractTarballToVolume(download.TarballPath(d.NodeConfig.KubernetesVersion, d.NodeConfig.ContainerRuntime), params.Name, BaseImage); err != nil {
+			glog.Infof("Unable to extract preloaded tarball to volume: %v", err)
+		} else {
+			glog.Infof("Took %f seconds to extract preloaded images to volume", time.Since(t).Seconds())
```

Comment:
anywhere else in minikube we have this format
"duration metric: took % to..." please use same format, for better search in logs expereince.

Reply:
done